website/docs/docs/dbt-versions/experimental-features.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Preview new and experimental features in dbt Cloud"
+id: "experimental-features"
+sidebar_label: "Preview new dbt Cloud features"
+description: "Gain early access to many new dbt Labs experimental features by enabling this in your profile."
+---
+
+dbt Labs often beta tests features before deciding to continue on the [Product lifecycle](https://docs.getdbt.com/docs/dbt-versions/product-lifecycles#dbt-cloud).
+
+You can access experimental features if you want to preview beta features that haven’t yet been released to dbt Cloud. You can toggle on or off all experimental beta features in one place. As with other beta features, these features:
+
+- May not be feature-complete or fully stable as we’re actively developing them.
+- Could be discontinued at any time.
+- May require feedback from you to understand their limitations or impact. Each experimental feature collects feedback directly in dbt Cloud.
```

Comment:
May help adoption to understand how we pick what moves forward in the dev process. 
```suggestion
- May require feedback from you to understand their limitations or impact. Each experimental feature collects feedback directly in dbt Cloud, which may impact dbt Labs' decisions to implement. 
```